quick_tour/the_big_picture.rst

```diff
@@ -29,12 +29,13 @@ When developing a Symfony application, your responsibility as a developer
 is to write the code that maps the user's *request* (e.g. ``http://localhost:8000/``)
 to the *resource* associated with it (the ``Homepage`` HTML page).
 
-The code to execute is defined in **actions** and **controllers**. The mapping
-between user's requests and that code is defined via the **routing** configuration.
-And the contents displayed in the browser are usually rendered using **templates**.
+The code to execute is defined in **controllers**  also called **actions** which
```

Comment:
The original description of actions/controllers is too short ... but the new is a bit confusing. What about the following?

```
The code to execute is defined as methods of PHP classes. The methods are
called **actions** and the classes **controllers**, but in practice most
developers use **controllers** to refer to both of them.
```